govcd/adminvdc_test.go

```diff
@@ -219,3 +219,48 @@ func (vcd *TestVCD) Test_UpdateVdcFlex(check *C) {
 	check.Assert(*updatedVdc.AdminVdc.IsElastic, Equals, true)
 	check.Assert(*updatedVdc.AdminVdc.IncludeMemoryOverhead, Equals, false)
 }
+
+// Tests VDC storage profile update
+func (vcd *TestVCD) Test_UpdateStorageProfile(check *C) {
```

Comment:
Name is misleading. Maybe something similar to:
```suggestion
func (vcd *TestVCD) Test_VDCUpdateStorageProfile(check *C) {
```

Reply:
changed